pkg/ratelimit/ratelimit.go

```diff
@@ -110,6 +110,9 @@ func (grl *GCERateLimiter) Accept(ctx context.Context, key *cloud.RateLimitKey)
 	return err
 }
 
+// Observe is a no-op func to satisfy cloud.RateLimiter
+func (*GCERateLimiter) Observe(context.Context, error, *cloud.RateLimitKey) {}
```

Comment:
```suggestion
func (grl *GCERateLimiter) Observe(context.Context, error, *cloud.RateLimitKey) {}
```

should we or is too much notpick?